pack/wsp.json

```diff
@@ -111,6 +111,7 @@
 		"deck_limit": 3,
 		"faction_code": "hero",
 		"flavor": "\"You're going to want to ice that in the morning.\" - Wasp",
+		"health_per_hero": false,
```

Comment:
What does this property do on an **event**?

Reply:
I removed this, thanks! I think in general, the `"field": false` stuff is unnecessary, especially if it doesn't apply to the card type.